addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -24,13 +24,21 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
 
   constructor() {
     super(...arguments);
+
     this.loadOptions();
-    this.loadProvidedValue();
+    // this.loadProvidedValue();
   }
 
   loadOptions() {
     const metaGraph = this.args.graphs.metaGraph;
-    const fieldOptions = this.args.field.options;
+    const fieldOptions = this.getFieldOptionsAsObject(this.args.field.options);
+
+    if (!this.isValidFieldOptions(fieldOptions)) {
+      console.error(`Options are invalid.`);
```

Comment:
nitpick: I would log this as part of the message in the isValidFieldOptions method since that one explains what the issue is as well.